app/src/main/kotlin/com/wire/android/util/ui/AssetImageFetcher.kt

```diff
@@ -0,0 +1,41 @@
+package com.wire.android.util.ui
+
+import android.content.res.Resources
+import android.graphics.drawable.BitmapDrawable
+import coil.ImageLoader
+import coil.decode.DataSource
+import coil.fetch.DrawableResult
+import coil.fetch.FetchResult
+import coil.fetch.Fetcher
+import coil.request.Options
+import com.wire.android.model.UserAvatarAsset
+import com.wire.android.util.toBitmap
+import com.wire.kalium.logic.feature.asset.GetAvatarAssetUseCase
+import com.wire.kalium.logic.feature.asset.PublicAssetResult
+
+class AssetImageFetcher(
+    val data: UserAvatarAsset,
+    val getAvatarAsset: GetAvatarAssetUseCase,
+    val resources: Resources,
+    val imageLoader: ImageLoader
```

Comment:
are we planning to use this or can we remove it?

Reply:
I'm not fully sure yet.

It depends on whether the custom-fetched images are Cached by Coil or not. My gut feeling is that no, they aren't.

But I don't want to solve caching in this PR. I'd leave it for a next one.